changelog_unreleased/html/pr-7556.md

```diff
@@ -0,0 +1,23 @@
+#### Format HTML attribute `style` value ([#7556](https://github.com/prettier/prettier/pull/7556) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```html
+<!-- Input -->
+<div style="   color : red;
+            display    :inline ">
+  </div>
+
+<!-- Prettier stable -->
+<div
+  style="   color : red;
+            display    :inline "
+></div>
+
+<!-- Prettier master -->
+<div
+  style="
+    color: red;
+    display: inline;
```

Comment:
I think `style` should be inlined

/cc @prettier/core What do you think?

Reply:
Yes.. I think inline is better if it can fit

Reply:
I guess this make sense. Will do